documentation/LLC-quickstart.md

```diff
@@ -0,0 +1,401 @@
+    package-version: 1.0.0-beta.1
+    rest-level-client: true
+    add-credentials: true
+    credential-scopes: "https://purview.azure.net/.default"
+    use-extension:
+      "@autorest/typescript": "6.0.0-beta.14"
+    ```
+    ~~~ 
+    Here, we need to replace the value in `package-name`, `title`, `description`, `input-file`, `package-version`, `credential-scopes` into **your own service's** `package-name`, `title`, `description` etc.   
+    
+    ---
+    **NOTE**
+    
+    It's always recommended to replace the version of code generator @autorest/typescript with the latest version you can find in [npmjs.com](https://www.npmjs.com/package/@autorest/typescript) in latest tag.  
+    
+    **For the first time, you will need to change the generate-metadata value as true, so that it can generate the basic package.json, tsconfig.json file for you. After the first generation, you can set it back as false as we have some manual changes in this file and don't want them get overwrite by generated ones.**   
+    
+    ---  
+  
+1. **edit rush.json**  
+    As the libraries in this azure-sdk-for-js repository are managed by rush, you need to add an entry in rush.json under projects section to make sure it works. For example:
+    ```
+        {
+          "packageName": "@azure-rest/purview-account",
+          "projectFolder": "sdk/purview/purview-account-rest",
+          "versionPolicyName": "client"
+        },
+    ```
+    Here you also need to replace the `packageName`, `projectFolder` into your own services'.
+    
+    ---  
+    **NOTE** 
+    
+    About the `versionPolicyName`, if the library you are working on is for data-plane, then it should be `client`, if the library you are working on is for control plane, then it should be `mgmt`.  
+    
+    ---  
+
+1. **run autorest to generate the SDK**  
+
+    Now you can run this command in swagger folder you just created. 
+    ```shell
+    autorest --typescript ./README.md
+    ```
+    After this finishes, you will see the generated code in src folder which are in the same level with the swagger folder.   
+    
+    
+    After that, you can get a workable package, and run the following commands to get a artifact if you like.
+    
+    ```shell
+    rush update
+    rush build -t <your-package-name>
+    cd <your-sdk-folder>
+    rushx pack
+    ``` 
+    But we still need to add some tests for it. 
+# How to write test for RLC
+In order to release it, we need to add some tests for it to make sure we are delivering high quality packages. but before we add the test, we need to manual change a few things to make the test framework works. 
+1. **update package.json file**  
+    Currently the generated will skip the actual test step. you should change it to make sure it works. 
+    First, change the `scripts` section from 
+    ~~~
+        "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
I think you also need to add these:

"build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
"build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
 "build:test": "tsc -p . && rollup -c 2>&1",